packages/backend/src/types.ts

```diff
@@ -0,0 +1,79 @@
+export interface BlueprintElement {
+    type: string;
+    properties?: Record<string, any>;
+}
+
+export interface Relationship extends BlueprintElement {
+    target: string;
+    // eslint-disable-next-line camelcase
```

Comment:
Optional:
`// eslint-disable-next-line camelcase` appears 8 times in this file - maybe it's worth to disable for the file 🤔 or group it and add: 
`
/* eslint-disable camelcase */
/* eslint-enable camelcase  */
`

Reply:
Since this file contains types for manager defined entities only I think it even makes sense to disable the rule for the whole file, which I did.